ecs_deploy/ecs.py

```diff
@@ -595,7 +595,6 @@ def update_task_definition(self, task_definition):
             volumes=task_definition.volumes,
             role_arn=task_definition.role_arn,
             execution_role_arn=task_definition.execution_role_arn,
-            tags=task_definition.tags,
```

Comment:
Hi @nitrocode, thanks for your PR!
Isn't this needed to keep the tags in the new task definition?

Best
Fabian

Reply:
Hi! Thanks for looking at it ☺️.

I think you're right. I'll step through this to make sure tags are passed appropriately